src/more-infos/more-info-climate.html

```diff
@@ -208,6 +238,20 @@
     }.bind(this), 500);
   },
 
+  computeTargetTempHidden: function (stateObj) {
+    return stateObj.attributes.operation_mode.includes('Off');
+  },
+
+  computeHideTempRangeSlider: function (stateObj) {
+    return !stateObj.attributes.operation_mode.includes('Auto');
+  },
+
+  computeHideTempSlider: function (stateObj) {
+    return (
```

Comment:
One changes the display on the state card. The other hides and displays the sliders on more infos if auto mode or off.